pyproject.toml

```diff
@@ -37,7 +38,8 @@ dependencies = [
   "polars>=0.18.3,<0.19.0",
   "pyarrow>=11.0.0,<12",
   "scipy>=1.5.4,<2",
-  "tqdm>=4.0.0,<5"
+  "tqdm>=4.0.0,<5",
+  "versioneer>=0.29"
```

Comment:
in non-vendor mode versioneer is only needed as a build dependency

Reply:
done